packages/storybook/.storybook/style.scss

```diff
@@ -1,6 +1,7 @@
 $formation-asset-path: '~@department-of-veterans-affairs/formation/assets';
 
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/uswds-typography.css";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-action-link.css";
```

Comment:
This import is temporary so that the visual changes can be tested in Storybook with the Formation styles still active. I will remove this import before merging.